Remove AltText module and fix container names

nomadicoder opened the pull request:
- Remove AltText module instead of upgradeing. Module is not
needed in latest versions of Omeka-S
- Use consistent container names - May fix some inter-
container communication problems.

nomadicoder committed:
Remove AltText module and fix container names

- Remove AltText module instead of upgradeing. Module is not
needed in latest versions of Omeak-S
- Use consistent container names - May fix some inter-
container communication problems.

sensei100 commented on Makefile:
```diff
@@ -20,6 +20,8 @@ MAIL_SERVER ?= "smtp.gmail.com"
 MAIL_ADDRESS ?= "omeka@example.com"
 MAIL_PASSWORD ?= replace_me
 
+.PHONEY: init-app
```
@nomadicoder I don't really understand what this is.  Can you explain it to me?

nomadicoder replied:
@sensei100 We were getting a notification in app that there was a version update to the AltText plugin.  When I went to get the update information to change to version number in the plugin, there was a note that the AltText plugin was no longer needed, as the feature had been added to the latest version of Omeka-S, which we're already using.  Gabe verified this.

I also added an unzip option so it would overwrite files without waiting for a prompt for when we do future upgrades.

nomadicoder replied:
@sensei100 and ensure we only clean up the zip files we downloaded.

nomadicoder replied:
I understand now.  Github should open the expansion automatically so it's not buried.

nomadicoder replied:
The `phony-init` was so the target gets triggered.  `make init-app` yielded `nothing to be done`.  However, it seems to be working without it.  I'll remove the definition.

Reference: https://unix.stackexchange.com/questions/403761/make-nothing-to-be-done-for-build:

nomadicoder replied:
Resolved for sure

nomadicoder commented:
The Makefile change was to resolve a problem running locally.  the database container was explicitly named `omeka-db`, and I thought that was the cause of a network problem.  It was actually a resource issue on my local machine.  But I thought I would keep the change so all the container names would be consistent.

nomadicoder committed:
Remove init-app target workaround

- `make init-app` seems to be working without the .PHONY declaration.
Go figure.

sensei100 commented on Makefile:
```diff
@@ -20,6 +20,8 @@ MAIL_SERVER ?= "smtp.gmail.com"
 MAIL_ADDRESS ?= "omeka@example.com"
 MAIL_PASSWORD ?= replace_me
 
+.PHONEY: init-app
```
@nomadicoder I am specifically asking about this:
.PHONEY: init-app

I don't see that being used anywhere, so I don't understand the purpose.

nomadicoder replied:
See above change.  I resolve it just before your last comment
